JustBrewIt/Pages/Index.cshtml.cs

```diff
@@ -1,7 +1,7 @@
 ﻿using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 using Microsoft.Extensions.Logging;
-using QuickType;
+using BreweryQuickType;
```

Comment:
Good to rename, but I'd drop the QuickType suffix.  Maybe this is what you can name BreweryAPI.

Reply:
updated namespace.